### DISCUSSION TITLE
Scrolling behaviour imporvemnt

### DISCUSSION THREAD

**Discussion by felixfeng33** [simonwep/viselect#233] [Q&A]
Currently, we are using `scrollTop` to simulate scrolling. According to the comments in the code, this is done to address the issue of inconsistent scrolling speeds across different browsers.However, the experience is not as smooth as the browser’s native scrolling; it feels like it occasionally stutters. Is it possible to retain the native scrolling while also solving the compatibility issues?

**Comment by simonwep** [comment]
It's been a very, very long time since I tackled this issue. You might be right, I'll have to take a look into this again. But if you have any idea of how this could be improved any help is highly appreciated, Iirc I spent quite the amount of time fixing this / making that part of the library work somehow consistent across browsers 😅

**Comment by felixfeng33** [comment]
After the scroll wheel becomes smooth, the selection area starts to lag. This seems to be a difficult trade-off.I forked this project to solve my issue, but it broke many other things. I solved it by setting the scroll area to absolute.But a lot of the concepts have changed.

**Comment by felixfeng33** [comment]
btw When the mouse moves and the mouse wheel operates at the same time, the wheel becomes almost unusable.
